openedx/adg/admin_dashboard/admin_settings/admin.py

```diff
@@ -1,3 +1,18 @@
 """
 Register models for `admin_settings` app
 """
+from django.contrib import admin
+
+from .models import Invitation
+
+
+@admin.register(Invitation)
+class InvitationAdmin(admin.ModelAdmin):
+    """
+    Django admin class for Invitation
+    """
+
+    fields = ('name', 'email', 'business_unit', 'permission', 'status',)
+    list_display = ('id', 'name', 'email', 'business_unit', 'permission', 'status',)
```

Comment:
If we are defining the fields here, let's keep the order the same as the intended order in the designs, i.e.:

email, status, permission, business unit